float32 err_trans
float32 err_rot

float32 w
float32 h

int32 plane_id